adoc/bp_chap_expanded_support.adoc

```diff
@@ -430,72 +428,97 @@ Some actions are not yet supported:
 // Reason: RMT and SMT cannot mirror Debian repositories (yet) and so cannot create it in that directory the correct files.
 // We are waiting for SMT to release the feature/fix to mirror Debian repositories. When this has been done, this comment and the limitation above can be removed.
 
+== Prepare {ubuntu} Clients for Registration
```

Comment:
Same comment as for beta, we do not prepare the  client, but the server to allow registration of these kind of clients.